name: "pick_and_place"
joint_target: [1.2, 0.0, 0.5, 0.0, 0.0, 0.0]
joint_home: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
dcart_target: [1.2, 0.0, 0.65, 0.0, 0.0, 0.0]
waypoints:
  - joint_angles: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    position: [0.1, 0.2, 0.3]
    orientation: [0.0, 0.0, 0.0]
    velocity: 50.0
  - joint_angles: [1.57, -0.5, 0.8, 0.0, 1.0, 0.0]
    position: [0.4, 0.5, 0.6]
    orientation: [0.1, 0.2, 0.3]
    velocity: 30.0
  - joint_angles: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    position: [0.1, 0.2, 0.3]
    orientation: [0.0, 0.0, 0.0]
    velocity: 50.0
  - joint_angles: [1.57, -0.5, 0.8, 0.0, 1.0, 0.0]
    position: [0.4, 0.5, 0.6]
    orientation: [0.1, 0.2, 0.3]
    velocity: 30.0
  - joint_angles: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    position: [0.1, 0.2, 0.3]
    orientation: [0.0, 0.0, 0.0]
    velocity: 50.0
